integrations/sveltekit/src/api-reference.ts

```diff
@@ -0,0 +1,36 @@
+import { getHtmlDocument } from '@scalar/core/libs/html-rendering'
+
+import { customTheme } from './custom-theme.js'
```

Comment:
should we use ts extension here?

```suggestion
import { customTheme } from './custom-theme.ts'
```